--- modules/uc_recurring_hosted/uc_recurring_hosted.module	2010-09-10 07:22:10.000000000 +0000
+++ modules/uc_recurring_hosted/uc_recurring_hosted.module.pending	2010-10-18 21:43:06.919485247 +0000
@@ -552,7 +552,6 @@ function uc_recurring_hosted_paypal_wps_
   // the recurring payment is setup at the time of product purchase
   if (!empty($_POST['subscr_id'])) {
     $fee->data['subscr_id'] = $_POST['subscr_id'];
-    uc_order_update_status($order->order_id, 'paypal_pending');
     return TRUE;
   }
   return FALSE;
